### PR TITLE
Update ec2-cloudwatch-metrics.md

### PR DESCRIPTION
**Pull request by nmohantyctr**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 1]
```diff
@@ -135,10 +137,24 @@ To configure a CloudTrail Source, perform these steps:
 2. [Configure DataEvents with CloudTrail](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/using-cloudtrail.html) in your AWS account.
 3. Confirm that logs are being delivered to the Amazon S3 bucket.
 4. Add an [AWS CloudTrail Source](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source.md) to Sumo Logic.
-5. While configuring the cloud trail log source, following Field can be added in the source:
-    1. Add an **account** field and assign it a value which is a friendly name / alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried via the “account field”.
-
+	1. **Name**. Enter a name to display the new Source.
+	2. **Description**. Enter an optional description.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:141]:
```suggestion
	2. **Description**.  You may skip the description as it's optional.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 2]
```diff
@@ -135,10 +137,24 @@ To configure a CloudTrail Source, perform these steps:
...
+	3. **S3 Region**. Select the Amazon Region for your API Gateway S3 bucket.
+	4. **Bucket Name**. Enter the exact name of your API Gateway S3 bucket.
+	5. **Path Expression**. Enter the string that matches the S3 objects you'd like to collect. You can use a wildcard `*` in this string.
+:::note
+DO NOT use a leading forward slash. See [Amazon Path Expressions](/docs/send-data/hosted-collectors/amazon-aws/Amazon-Path-Expressions). The S3 bucket name is not part of the path. Don’t include the S3 bucket name when you are setting the Path Expression.
+:::
+	6. **Source Category**. Enter aws/observability/cloud trail/logs.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:148]:
```suggestion
	6. **Source Category**. Enter `aws/observability/cloud trail/logs`.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 3]
```diff
@@ -135,10 +137,24 @@ To configure a CloudTrail Source, perform these steps:
...
+	7. **Fields**. Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried through the “account field”.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:149]:
```suggestion
	7. **Fields**. Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried through the **account field**.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 4]
```diff
@@ -135,10 +137,24 @@ To configure a CloudTrail Source, perform these steps:
...
 <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-EC2-CW-Metrics/Fields.png')} alt="Fields" />
+	8. **Access Key ID and Secret Access Key**. Enter your [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html). Learn how to use Role-based access to AWS [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+	9. **Log File Discovery -> Scan Interval**. Use the default of 5 minutes. Alternately, enter the frequency. Sumo Logic will scan your S3 bucket for new data. Learn how to configure Log File Discovery [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+	10. **Enable Timestamp Parsing**. Select the check box.
+	11. **Time Zone**. Select Ignore time zone from the log file and instead use, and select UTC.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:154]: Lines 154, 155: Could you please recheck and confirm if we are conveying the correct instructions?

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 5]
```diff
@@ -135,10 +137,24 @@ To configure a CloudTrail Source, perform these steps:
...
+	12. **Timestamp Format**. Select Automatically detect the format.
+	13. **Enable Multiline Processing**. Select the check box, and select Infer Boundaries.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:156]:
```suggestion
	13. **Enable Multiline Processing**. Select the check box and select Infer Boundaries.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 6]
```diff
@@ -262,7 +278,7 @@ Use this dashboard to:
 * Monitor top IAM Users, Assumed Role Users and User agents
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:278]:
```suggestion
* Monitor top IAM Users, Assumed Role Users, and User agents
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting changes

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 7]
```diff
@@ -117,14 +117,16 @@ The Sumo Logic App for AWS EC2 (CloudWatch Metrics) allows you to collect your E
...
+
+* Sumo Logic supports collecting metrics using two source types
+	* Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)
+		Or
+	* Configure an [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
+:::note
+Namespace for **Amazon EC2** Service is **AWS/EC2**.
+:::
+	* **Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried through the “account” field.
 <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-EC2-CW-Metrics/Metadata+account.png')} alt="Metadata" />
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:130]: Please ensure the alignment here looks ok by running a `yarn start`

**Review by JainM6 — APPROVED** [review]

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 8]
```diff
@@ -117,14 +117,16 @@ The Sumo Logic App for AWS EC2 (CloudWatch Metrics) allows you to collect your E
...
+### Collect Amazon CloudWatch EC2 Metrics
+
+* Sumo Logic supports collecting metrics using two source types
+	* Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)
+		Or
+	* Configure an [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
+:::note
+Namespace for **Amazon EC2** Service is **AWS/EC2**.
+:::
+	* **Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried through the “account” field.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:129]:
```suggestion
Sumo Logic supports collecting metrics using two source types:
* Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) **(recommended)**
* Configure an [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
:::note
Namespace for **Amazon EC2** Service is **AWS/EC2**.
:::

* **Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried through the “account” field.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 9]
```diff
@@ -135,10 +137,24 @@ To configure a CloudTrail Source, perform these steps:
...
 <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-EC2-CW-Metrics/Fields.png')} alt="Fields" />
+	8. **Access Key ID and Secret Access Key**. Enter your [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html). Learn how to use Role-based access to AWS [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+	9. **Log File Discovery -> Scan Interval**. Use the default of 5 minutes. Alternately, enter the frequency. Sumo Logic will scan your S3 bucket for new data. Learn how to configure Log File Discovery [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+	10. **Enable Timestamp Parsing**. Select the check box.
+	11. **Time Zone**. Select Ignore time zone from the log file and instead use, and select UTC.
+	12. **Timestamp Format**. Select Automatically detect the format.
+	13. **Enable Multiline Processing**. Select the check box and select Infer Boundaries.
+	14. Click Save.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:157]:
```suggestion
5. **Source Category**. Enter `aws/observability/cloud trail/logs`.
6. **Fields**. Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried through the **account** field. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-EC2-CW-Metrics/Fields.png')} alt="Fields" />
7. **Access Key ID and Secret Access Key**. Enter your [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html). Learn how to use Role-based access to AWS [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
8. **Log File Discovery -> Scan Interval**. Use the default of 5 minutes. Alternately, enter the frequency. Sumo Logic will scan your S3 bucket for new data. Learn how to configure Log File Discovery [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
9. **Enable Timestamp Parsing**. Select the check box.
10. **Time Zone**. Select Ignore time zone from the log file and instead use, and select UTC.
11. **Timestamp Format**. Select Automatically detect the format.
12. **Enable Multiline Processing**. Select the check box and select Infer Boundaries.
13. Click **Save**.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 10]
```diff
@@ -135,10 +137,23 @@ To configure a CloudTrail Source, perform these steps:
...
-
-<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-EC2-CW-Metrics/Fields.png')} alt="Fields" />
+	1. **Name**. Enter a name to display the new Source.
+	2. **Description**.  You may skip the description as it's optional.
+	3. **S3 Region**. Select the Amazon Region for your API Gateway S3 bucket.
+	4. **Bucket Name**. Enter the exact name of your API Gateway S3 bucket.
+	5. **Path Expression**. Enter the string that matches the S3 objects you'd like to collect. You can use a wildcard `*` in this string.
+:::note
+DO NOT use a leading forward slash. See [Amazon Path Expressions](/docs/send-data/hosted-collectors/amazon-aws/Amazon-Path-Expressions). The S3 bucket name is not part of the path. Don’t include the S3 bucket name when you are setting the Path Expression.
+:::
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:147]:
```suggestion
   1. **Name**. Enter a name to display the new Source.
   2. **Description**.  You may skip the description as it's optional.
   3. **S3 Region**. Select the Amazon Region for your API Gateway S3 bucket.
   4. **Bucket Name**. Enter the exact name of your API Gateway S3 bucket.
   5. **Path Expression**. Enter the string that matches the S3 objects you'd like to collect. You can use a wildcard `*` in this string.
   :::note
   DO NOT use a leading forward slash. See [Amazon Path Expressions](/docs/send-data/hosted-collectors/amazon-aws/Amazon-Path-Expressions). The S3 bucket name is not part of the path. Don’t include the S3 bucket name when you are setting the Path Expression.
   :::
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md (hunk 3 above)
Comment (kimsauce) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md]:
```suggestion
	7. **Fields**. Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried through the **account** field.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 11]
```diff
@@ -135,10 +137,23 @@ To configure a CloudTrail Source, perform these steps:
...
+   :::
+5. **Source Category**. Enter `aws/observability/cloud trail/logs`.
+6. **Fields**. Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried through the **account** field. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-EC2-CW-Metrics/Fields.png')} alt="Fields" />
+7. **Access Key ID and Secret Access Key**. Enter your [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html). Learn how to use Role-based access to AWS [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+8. **Log File Discovery -> Scan Interval**. Use the default of 5 minutes. Alternately, enter the frequency. Sumo Logic will scan your S3 bucket for new data. Learn how to configure Log File Discovery [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+9. **Enable Timestamp Parsing**. Select the check box.
+10. **Time Zone**. Select Ignore time zone from the log file and instead use, and select UTC.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:153]:
```suggestion
10. **Time Zone**. Select **Ignore time zone** from the log file and instead select UTC.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 12]
```diff
@@ -135,10 +137,23 @@ To configure a CloudTrail Source, perform these steps:
...
+7. **Access Key ID and Secret Access Key**. Enter your [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html). Learn how to use Role-based access to AWS [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+8. **Log File Discovery -> Scan Interval**. Use the default of 5 minutes. Alternately, enter the frequency. Sumo Logic will scan your S3 bucket for new data. Learn how to configure Log File Discovery [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+9. **Enable Timestamp Parsing**. Select the check box.
+10. **Time Zone**. Select **Ignore time zone** from the log file and instead select UTC.
+11. **Timestamp Format**. Select Automatically detect the format.
+12. **Enable Multiline Processing**. Select the check box and select Infer Boundaries.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:155]:
```suggestion
12. **Enable Multiline Processing**. Select the check box and select **Infer Boundaries**.
```

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md [hunk 13]
```diff
@@ -117,14 +117,16 @@ The Sumo Logic App for AWS EC2 (CloudWatch Metrics) allows you to collect your E
...
-
-* **Metadata:** Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account field”.
+Sumo Logic supports collecting metrics using two source types:
+* Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) **(recommended)**
+* Configure an [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
+:::note
+Namespace for **Amazon EC2** Service is **AWS/EC2**.
+:::
 
+* **Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried through the “account” field.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md:129]:
```suggestion
* **Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried through the **account** field.
```

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
@nmohantyctr , can you please respond to @JainM6 's [question](https://github.com/SumoLogic/sumologic-documentation/pull/1524#discussion_r1092119992)?

File: docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md (hunk 4 above)
Comment (kimsauce) [reply docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md]: Hi Megha, as these steps are present in 44 apps docs total, [I've created a separate ticket for Nishant](https://app.asana.com/0/1183741177809740/1203999216349981/f) to address this with the Apps and AWSO teams.

**Pull request merged**